### DISCUSSION TITLE
Strange static during recordings. New install on RPi4

### DISCUSSION THREAD

**Discussion by Billium28** [mcguirepr89/BirdNET-Pi#861] [General]
I redid my birdnet with a new Raspberry Pi 4 and a UGreen USB sound card and a BY-M1 microphone. My previous build was the same mic and usb card but with a Libre Renegade. My mic would not pick up much, everything was extremely quiet. So I am trying now with an RPi4. The issue is I now get this terrible electronic screeching instead of real audio. Below is a picture of the spectrogram. 

![Screenshot_20230418_065102_Firefox](https://user-images.githubusercontent.com/11811885/232757087-31b7308e-5810-467d-b10a-2ef2b6a14db4.jpg)

I would upload a sound file but I'm not sure how to here.

**Comment by lloydbayley** [comment]
Yeh, that looks bad and I used to get it when there was a problem with the USB mic. Are you able to test the mic/soundcard on another machine (say, a laptop) just to make sure a fault hasn't developed or perhaps there's an intermittent one? In my case, many moons ago, the mic itself had a nasty solder joint that I re-flowed and it's now working well. See if you can do a test first.
